misc/services/camel/de-metas-camel-externalsystems/core/src/main/java/de/metas/camel/externalsystems/core/AppConfiguration.java

```diff
@@ -61,19 +64,44 @@ public void auditEventNotifier()
 				.addEventNotifier(new AuditEventNotifier(producerTemplate()));
 	}
 
+	@PostConstruct
+	public void authorizationTokenNotifier()
+	{
+		final String defaultAuthToken = context.getEnvironment().getProperty(ExternalSystemCamelConstants.MF_API_AUTHORIZATION_TOKEN_PROPERTY);
+		// note that calling producerTemp8late() here does *not* mean wh create an additional instance. See https://docs.spring.io/spring-framework/docs/current/reference/html/core.html#beans for details
```

Comment:
also, instead of injecting the `context`, inject only the `MetasfreshAuthProvider`... since this in on a `postContruct ` handler, the bean should be there

Reply:
also, I don't think we need the producer template here as the `MetasfreshAuthorizationTokenNotifier` should not be calling any routes